mattermost-plugin/go.mod

```diff
@@ -2,16 +2,10 @@ module github.com/mattermost/focalboard/mattermost-plugin
 
 go 1.18
 
-replace github.com/mattermost/focalboard/server => ../server
-
-replace github.com/mattermost/mattermost-server/v6 => github.com/mattermost/mattermost-server/v6 v6.0.0-20220613202234-182ae1234a49
-
 require (
 	github.com/golang/mock v1.6.0
-	github.com/mattermost/focalboard/server v0.0.0-00010101000000-000000000000
-	github.com/mattermost/mattermost-plugin-api v0.0.27
-	github.com/mattermost/mattermost-server/v6 v6.3.0
-	github.com/stretchr/testify v1.7.1
+	github.com/mattermost/mattermost-plugin-api v0.0.28-0.20220623051512-0afd85e854d4
```

Comment:
if we are deprecating the plugin api, can we remove this dependency?

Reply:
The only place left using it is in the OnActivate method (plugin.go) which is mostly gutted.  I left it in there as it initializes a connection to the master DB and I didn't duplicate that code.  Most setup logic is now in the boards package.